k8s/aat/common/sscs/sscs-case-loader.yaml

```diff
@@ -0,0 +1,27 @@
+---
```

Comment:
I think this should go into both clusters and not common with different schedules?